website/siteConfig.js

```diff
@@ -67,6 +66,7 @@ const sponsors = [
         total: sponsor.totalDonations,
       };
     }),
+  ...sponsorsManual,
```

Comment:
Need to override since duplicate entries (in this case coinbase previous donated on OC earlier/stopped and it's different)